source/user/lo.rst

```diff
@@ -0,0 +1,66 @@
+.. _user_lo:
+
+**********************
+Localized Orbitals
+**********************
+
+*Modules*: :mod:`lo`
+
+Introduction
+============
+A molecular orbital is usually delocalized, i.e. its spatial distribution is
+all over the whole molecular rather than localized around several atoms or bonds.
```

Comment:
```suggestion
A molecular orbital is usually delocal, i.e. it has non-negligible amplitude all over the whole system rather than only around a specific region localized around some atom(s) or bond(s).
```